### ISSUE TITLE
Only Strict Search working. Lenient Search not working properly.

### ISSUE THREAD

**Issue by bakarep** [meilisearch/MeiliSearch#1618]
I have text in first title as "ABC Status LEDs for Ciyana Equipment used with XYZ Service"
And in second title there is text in between as "LED status".

When I provide search string as "LED status", only second title is returned. Why is first not returned?
When I search for "Ciyana LED" only first title is returned. Why is second not returned, even though it has LED in it?
When I search for "Ciyana LED status" NOTHING is returned. Why are both the titles not returned, even though some search words match in it?

In short , why is Meilisearch so strict? Is there a setting to make sure it is NOT that strict in matching and shows results that are matching any of the keywords from search string ?

**Comment by curquiza** [comment]
Closing this issue since the answer has been given 🙂
